src/single-movie/SingleMovie.js

```diff
@@ -8,21 +8,29 @@ export default class SingleMovie extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      singleMovie: {},
+      singleMovie: {
+        title: '',
+        backdrop_path: '',
+        tagline: '',
+        average_rating: 0,
+        overview: '',
+        release_date: '',
+        genres: [],
+        budget: '',
+        revenue: 0,
+        runtime: ''
+      },
```

Comment:
setting the initial values to be rendered will help in the case of something not showing